cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesController.java

```diff
@@ -505,6 +506,21 @@ public void getAll(@NotNull Context ctx) {
             }
             addDeprecatedContentTypeWarning(ctx, contentType);
             requestResultSize.update(results.length());
+        } catch (DataAccessException ex) {
```

Comment:
This should be handled globally as unit conversion issues are not just a timeseries concern. The NotFoundException/IllegalArgumentExceptions are not examples to follow as they were implemented prior to the common code added to the base class.